backend/homework/4/src/main/java/org/example/question1/Main.java

```diff
@@ -0,0 +1,42 @@
+package org.example.question1;
+
+import org.example.LoggerFile;
+
+import java.util.Arrays;
+
+public class Main {
+
+    private static final LoggerFile log = new LoggerFile();
```

Comment:
Look into how logger custom methods can be invoked without initiating objects in every class.